### PR TITLE
feat:flash message on login, on cake5

### PR DESCRIPTION
**Pull request by arodu**
feat https://github.com/CakeDC/users/issues/994

### CODE REVIEW

File: config/users.php [hunk 1]
```diff
@@ -47,6 +47,9 @@
             // determines if registration workflow includes email validation
             'validate' => true,
         ],
+        'Login' => [
+            'flashMessage' => false, // bool
```
Comment (arodu) [inline config/users.php:51]: @rochamarcelo Do you think we should leave it as false by default? or do we activate it at once?

Comment (rochamarcelo) [reply config/users.php:51]: false is good here

**Comment by rochamarcelo** [comment]
Please fix conflicts, then it will be okay to merge.

**Comment by rochamarcelo** [comment]
Thank You.

**Pull request merged**